### ISSUE TITLE
Vscode ALT cannot eject keyboard circles face the tools menu

### ISSUE THREAD

**Issue by dingpengyu** [Microsoft/vscode#37461]
Hello, I am a vscode for blind users, when I use vscode, found some small problems, when I opened the vscode ALT case, f file pop-up menu, then vs cannot be opened normally, then I can only use alt+f f to open the file menu, though open, but can only see a small part of the complete tool menu will not see, I hope can fix this bug?
The following is my operating environment and the ways to achieve:
Screen reader:
NVDA
Implementation steps:
First, open the vscode, then the ALT on the keyboard, if you can't pop up the file f menu, then you can use alt+f to open a part

- VSCode Version:VSCodeSetup-x64-1.18.0-insider
- OS Version:windows1064bit-1709

**Comment by bpasero** [comment]
@dingpengyu I am trying to understand if this issue has already been reported before. are you saying that you press Alt key, then release it and then press F key and the file menu does not show up?
